src/index.js

```diff
@@ -45,6 +46,7 @@ const HANDLERS = {
   'experimentation-ess-daily': essExperimentationDaily,
   'experimentation-ess-all': essExperimentationAll,
   costs,
+  'gsc-pdp-structured-data': gscPdpStructuredData,
```

Comment:
I would have seen this as a generic `structured-data` audit, whereas the pages to which it's applied and the source is internal implementation details.

Reply:
Doesn't the current Audit Builder structure tie the implementation details for the source to the audit itself? As you might want different schedules for different sources for example that could be an issue. But I see your point, we can discuss that another time and build a generic first iteration here